File: modules/x-engine-native-viewer-orgi/android/native-viewer-orgi/src/main/java/com/zkty/nativ/viewer-orgi/Nativeviewer-orgi.java
package com.zkty.nativ.viewer-orgi;

import android.util.Log;

import com.tencent.mmkv.MMKV;
import com.zkty.nativ.core.NativeModule;
import com.zkty.nativ.core.XEngineApplication;

import java.io.File;

public class Nativeviewer-orgi extends NativeModule implements Iviewer-orgi {

    @Override
    public String moduleId() {
        return "com.zkty.native.viewer-orgi";
    }

    @Override
    public void afterAllNativeModuleInited() {

    }
}

File: modules/x-engine-native-viewer-orgi/iOS/Class/Native_viewer-orgi.m
//
//  Native_viewer-orgi.m
//  viewer-orgi
//
//  Created by zk on 2020/9/7.
//  Copyright © 2020 edz. All rights reserved.


#import "Native_viewer-orgi.h"
#import "NativeContext.h"

@interface Native_viewer-orgi()
{ }
@end

@implementation Native_viewer-orgi
NATIVE_MODULE(Native_viewer-orgi)

 - (NSString*) moduleId{
    return @"com.zkty.native.viewer-orgi";
}

- (int) order{
    return 0;
}

- (void)afterAllNativeModuleInited{
} 
-(NSString*) test{
    return @"test";
}
@end
 
